### DISCUSSION TITLE
Is this bad?

### DISCUSSION THREAD

**Discussion by alexkess101** [react-hook-form/react-hook-form#3803] [Q&A] [answered]
`onClick={() => hookForm.formState.touched[name] = true}`

I'm manually changing the touched state of a few inputs. Everything works properly I'm just wondering if there is anything I'm missing that could potentially be a problem.

**Comment by bluebill1049** [comment, marked as answer]
It's a hack which may cause issue, you want to submit a feature request to support set dirty with `setValue` API?
